Formula/odpi.rb

```diff
@@ -0,0 +1,33 @@
+class Odpi < Formula
+  desc "Oracle Database Programming Interface for Drivers and Applications"
+  homepage "https://oracle.github.io/odpi/"
+  url "https://github.com/oracle/odpi/archive/v2.1.0.tar.gz"
+  sha256 "b74e06feb1a25d907b97739eda500d03af50c87cd5861b27a0e0baf79f739c06"
+
+  def install
+    system "make"
+
+    lib.install Dir["lib/*"]
```

Comment:
It seems there is no `make install` target. Could you open an issue upstream to ask them to support that? `make install PREFIX=...` would be greatly useful (and also include documentation, etc.)

Reply:
According to README Oracle prefers embedding ODPI-C source over providing as shared lib. Makefile is just a sample how to build it but that's it.
I will open an issue but I doubt it will be fixed soon.